tremor-connectors/src/impls/http/server.rs

```diff
@@ -399,18 +419,17 @@ impl Sink for HttpServerSink {
             if let Some(rid) = http_meta.get_u64("request_id").map(RequestId::new) {
                 match response_map.entry(rid) {
                     Entry::Vacant(k) => {
-                        if let Some((rid, sender)) = self.inflight.remove(&rid) {
+                        if let Some(_snot) = self.inflight.get(&rid) {
```

Comment:
I'm not sure how significant this is given that we have no tests whatsoever that asserts any streaming behaviour and the change is isolated to an event batch. So not sure how this can be a regression as there's nothing asserting the behaviour in question. Maybe we need a better term than streaming but I see what you mean!

Reply:
Added a note to revisit if/when we get around to SSE or streaming batch responses